bindings_ffi/src/mls.rs

```diff
@@ -1046,11 +1048,13 @@ mod tests {
         }
     }
 
-    pub struct MockLogger {}
+    pub struct MockLogger {
+        inbox_id: String,
```

Comment:
Nice change. Makes life much easier.

Reply:
It actually doesn't work currently :( There is one global MockLogger used, and the first one wins - so the second inbox ID in tests just logs with the first inbox's logger. We can probably cook up some clever way to log inbox ID's by default in the future, but it can't be done by storing it on the logger.